### DISCUSSION TITLE
A Comprehensive Guide to Core Coin’s Denomination Units | Core Blog

### DISCUSSION THREAD

**Discussion by giscus** [core-coin/web-blog#19] [General]
# A Comprehensive Guide to Core Coin’s Denomination Units | Core Blog

Core Blockchain Blog

https://blog.coreblockchain.cc/posts/blockchain/denomination-units.html

**Comment by PacoFi8** [comment]
I love it, this is a huge, nice explained in details answered a lots of questions!
"Voila", 
the digital economy is revealed and now step by step people start to understand 
what importance the DApp's CoreGroup.cc platform has. 

XCB ₡
Awesome guys

**Reply by PacoFi8** [reply]
without the a between is and huge!!!!
